app/components/topicPicker.js

```diff
@@ -67,66 +80,34 @@ export default React.createClass({
     // TODO: break this into its component pieces
     // const {topicId}: string = this.context.router.getCurrentParams();
 
+    console.log(this.props);
+
     return (
       <div className='topic-picker topic-picker--left'>
         <ul className='topic-picker__nav'>
-          <li data-topic='1'><b>Topic 1</b></li>
-          <li data-topic='2'><b>Topic 2</b></li>
-          <li data-topic='3'><b>Topic 3</b></li>
-          <li data-topic='4'><b>Topic 4</b></li>
+          { this.props.topicsTmp.map(topic => {
```

Comment:
cool. i've got a branch out that has starts to use the actions / store stuff for this so if we get these two merged, i can then merge master into my branch, resolve any conflicts and we'll have a good pattern for going forward.